scripts/GetClrDbg.sh

```diff
@@ -1,15 +1,42 @@
 #! /bin/bash
 
-script_dir=`dirname $0`
+script_dir="$( cd $( dirname "$0" ); pwd )"
+
+# ClrDbg Meta Version. It could be something like 'latest', 'vs2015', or a fully specified version. 
+__ClrDbgMetaVersion=
+
+# Install directory of the clrdbg relative to the script. 
+__InstallLocation=
+
+# When SkipDownloads is set to true, no access to internet is made.
+__SkipDownloads=false
+
+# Launches ClrDbg after downloading/upgrading.
+__LaunchClrDbg=false
+
+# Removes existing installation of ClrDbg in the Install Location.
+__RemoveExisting=false
+
+# Internal, fully specified version of the ClrDbg. Computed when the meta version is used.
+__ClrDbgVersion=
 
 print_help()
 {
-    echo 'GetClrDbg.sh <version> [<install path>]'
+    echo 'GetClrDbg.sh [-rsdh] -v V [-l L]'
     echo ''
     echo 'This script downloads and configures clrdbg, the Cross Platform .NET Debugger'
-    echo 'If <install path> is not specified, clrdbg will be installed to the directory'
-    echo 'from which this script was executed.' 
-    echo '<version> can be "latest" or a version number such as 14.0.25109-preview-2865786'
+    echo '-r    Deletes the existing installation of the debugger at the installation location. Can be used for a clean install of the debugger.'
```

Comment:
The '-r' argument doesn't seem to act quite like what I was thinking. I could definitely see this being useful for some scenarios, but the one I think we need is 'clean the old directory on upgrade'.